sips/adapt_fees_to_declared_effective_balance.md

```diff
@@ -0,0 +1,198 @@
+| Author | Title          | Category                             | Status  | Date               |
+|-------|----------------|--------------------------------------|---------|--------------------|
+| Gal Rogozinski & Marco Tabasco | Adapt fees to declared effective balance | Contracts & Core | open-for-discussions  |18.09.2024 |
+
+
+## Summary
+
+Creates a mechanism to incentivize users to record the correct balance of their validator with the contract.
+This balance will be used for operator and network fee calculations.
+
+## Motivation
+
+Operators on SSV Network charge validators a fee based on an expected workload. The expected workload is proportional to the the effective balance a validator has. Currently the effective balance is assumed to be a constant 32 ETH by the contract. With the introduction of [EIP-7251](https://eips.ethereum.org/EIPS/eip-7251) the effective balance of a validator could be as high as 2048 ETH.
```

Comment:
The workload won't increase linearly in proportion to eff balance... so maybe this should be rephrased